.travis/deploy.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
Is there some documentation about this step?

Reply:
exactly $DEPLOY_KEY, $DEPLOY_HOST and $DEPLOY_PATH

Reply:
This is custom deployment?

Reply:
This script is only executed if a deploy is requested. It scp's artifacts to `$DEPLOY_HOST:$DEPLOY_PATH` using the base64-decoded `$DEPLOY_KEY`. I wrote it to test the output of my jobs.

On the main repo it could be used to copy binaries to the ResidualVM downloads server.
Alternatively, we could store the results in Amazon S3 or Google Cloud buckets, but that is separate from the current infrastructure and not free.

Reply:
I'm interesting at this point only upload Android builds for daily snapshots. But rest use for CI as addition to Buildbot. Or use different folder for Travis builds and upload all. I think last part make more sense at this point.

Reply:
> I'm interesting at this point only upload Android builds for daily snapshots.

I personally would prefer all daily builds to come from Travis. The toolchains and build configuration used by Travis are fully open source and make it possible for anybody to make improvements. On the other hand, buildbot has been broken / provided suboptimal builds multiple times in the past because only very few people can make changes to its configuration.

Reply:
Daily or CI builds could be hand off to Travis where possible to save CPU time. Other builds still be here. It need to tuned to have both on the same folder. The builds where not broken on buildbot, but commits broke MacOS and AmigaOS targets. Current Jenkins builds are limited and allow merge broken commits. Linux builds where switched off at some point due host transfer. I wanted to enabled them again, but I forgot it about :( Now started changing Linux targets and moving into Travis at some point. Also I want update AmigaOS target. Windows daily builds are/will be provided manual way as usually. What I don't know yet how to make Travis CI and daily builds. As daily builds are not enough. Also current Travis builds are very limited to targets. Anyway I'm going to merge this PR.